packages/story-editor/src/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -88,13 +109,14 @@ function PaginatedMediaGallery({
       setNextPage();
       return;
     }
-
-    // Load the next page if the page isn't full, ie. scrollbar is not visible.
-    if (node.clientHeight === node.scrollHeight) {
-      setNextPage();
-      return;
-    }
-  }, [resources, hasMore, isMediaLoaded, isMediaLoading, setNextPage]);
```

Comment:
@samwhale  this is what broke the media3p test. It wasn't that the test are dependent on one another. Because in ci the screen is set to 1600:1000 meaning that the media3p video tab needs to make 2 api calls to get enough items to fill the container. 3pMedia is set to only gran 20 items, not 50 like local is. Since I ripped this out, it broke your test because with a screen that size, the second api call wasn't ever made. Additionally that meant that you couldn't ever scroll down to get more items. I fixed this by adding the debounced callback and then checking again if we actually still need the second api call.

Reply:
OOF my bad. Great find - that is super duper tricky. I owe you one 😓

Reply:
If that comment hadn't of been left I would have glossed right over it. 🙌🏽 Go team.